src/components/ActionList/components/Item/Item.tsx

```diff
@@ -40,30 +41,30 @@ export function Item({
   let prefixMarkup: React.ReactNode | null = null;
 
   if (prefix) {
-    prefixMarkup = <div className={styles.Prefix}>{prefix}</div>;
+    prefixMarkup = <span className={styles.Prefix}>{prefix}</span>;
   } else if (icon) {
     prefixMarkup = (
-      <div className={styles.Prefix}>
+      <span className={styles.Prefix}>
         <Icon source={icon} />
-      </div>
+      </span>
     );
   } else if (image) {
     prefixMarkup = (
-      <div
+      <span
         role="presentation"
         className={styles.Prefix}
         style={{backgroundImage: `url(${image}`}}
-      />
+      ></span>
```

Comment:
```suggestion
      />
```
Tests are failing because this can be a self closing tag